public/css/style.css

```diff
@@ -28,6 +28,13 @@
     height: 150px;
 }
 
+.selected-image-display {
+    margin: 1px 1px -3px 1px;
+    padding: 0 0 0 0;
+    width: 300px;
+    height: 300px;
+}
+
```

Comment:
Ok, we can leave this to later in the development process. Could you add an issue to change it to % to make sure we track this?

My preferred min res is 840, because it allows people to have two browser windows side-by-side if they have 1680 horizontal resolution. I realise it might be a bit low — what do you think? Anyway move this discussion to the issue itself. =)